src/body.rs

```diff
@@ -2,7 +2,78 @@
 //!
 //! This module includes types like `Json`, which can be used to automatically (de)serialize bodies
 //! using `serde_json`.
-
+//!
+//! # Examples
+//!
+//! Read/Write `Strings` and `Bytes` from/to bodies:
+//!
+//! ```rust, no_run
+//! # #![feature(async_await, futures_api)]
+//! use tide::body;
+//!
+//! async fn echo_string(msg: body::Str) -> String {
+//!     println!("String: {}", *msg);
+//!     format!("{}", *msg)
+//! }
+//!
+//! async fn echo_string_lossy(msg: body::StrLossy) -> String {
+//!     println!("String: {}", *msg);
+//!     format!("{}", *msg)
+//! }
+//!
+//! async fn echo_vec(msg: body::Bytes) -> String {
```

Comment:
Why not return a `Vec<u8>` here?

Reply:
Thank you!

Returning a `Vec<u8>` is not working for me...maybe because there's no `IntoResponse` defined for it?

Reply:
that is probably it - that should be pretty easy to add though. You can see (here)[https://github.com/rust-net-web/tide/blob/master/src/response.rs#L27] we are turning a string into a `Vec<u8>` anyways :P

Reply:
I'll add it :+1: But I think it would be cleaner to create a separate PR since it has nothing to do with #17 - thoughts?

Reply:
I don't mind it being in the same PR, it can still be a separate commit.

Reply:
Ok, added https://github.com/rust-net-web/tide/pull/97/files#diff-6cfc163228cc31bed4a7214fa0b69300R23 and updated the examples for body types.